linera-core/src/client/mod.rs

```diff
@@ -2124,6 +2125,14 @@ where
         let committee = self.local_committee().await?;
         let max_size = committee.policy().maximum_block_proposal_size;
         block.check_proposal_size(max_size)?;
+        for blob in &blobs {
+            if published_blob_ids.contains(&blob.id()) {
+                committee
+                    .policy()
+                    .check_blob_size(blob.content())
```

Comment:
Is there a global limit for published blobs' size? I.e. not only the per-blob size limit but also sum of all blobs published as part of the single proposal

Reply:
Only implicitly: There is a maximum blob size and a maximum number of published blobs per block. So the product of them is the maximum total size.